lib/services/local/presence.go

```diff
@@ -798,20 +798,20 @@ func (s *PresenceService) DeleteAllRemoteClusters() error {
 	return trace.Wrap(err)
 }
 
+// this combination of backoff parameters leads to worst-case total time spent
+// in backoff between 1200ms and 2400ms depending on jitter.  tests are in
+// place to verify that this is sufficient to resolve a 20-lease contention
+// event, which is worse than should ever occur in practice.
+const baseBackoff = time.Millisecond * 300
+const leaseRetryAttempts int64 = 6
```

Comment:
Not sure if it is worth doing now, but it would be nice if these were variables in PresenceService so we could add test coverage for the LimitExceeded error without having to wait 4.5 seconds.

Reply:
Yeah I agree. I was trying to limit the noise in this PR though.